packages/nimble-components/src/mapping/empty/index.ts

```diff
@@ -0,0 +1,28 @@
+import { DesignSystem } from '@microsoft/fast-foundation';
+import { attr } from '@microsoft/fast-element';
+import { Mapping } from '../base';
+import { template } from '../base/template';
+import type { MappingKey } from '../base/types';
+
+declare global {
+    interface HTMLElementTagNameMap {
+        'nimble-mapping-empty': MappingEmpty;
+    }
+}
+
+/**
+ * Maps data values to text.
+ * One or more may be added as children of a nimble-table-column-mapping element. The
+ * mapping displays an empty cell and text-only group rows.
+ */
+export class MappingEmpty extends Mapping<MappingKey> {
+    @attr()
+    public text?: string;
```

Comment:
The `group-row-placeholder-empty` label is already used for values that are unmapped values. (i.e. enum mappings for 1,2,3 but you assign value 4). If you have a mapped value that you want to render blank I think providing a name makes sense, otherwise why bother?

> would probably rather we default the value of text to '',

Specifically to this implementation point, only boolean attributes should be assigned a value and that value is false. FAST will try to update the attribute to `text=""` and that can cause races with someone trying to set the attribute at the same time. 

If there is a use-case not being covered for allowing a mapping with the label not specified, we should use a different implementation strategy.

Reply:
> The group-row-placeholder-empty label is already used for values that are unmapped values. (i.e. enum mappings for 1,2,3 but you assign value 4).

That actually isn't the current behavior. We decided against having a default placeholder label on group rows for the mapping columns because we said that having values that didn't correspond to any mapping's key was incorrect usage of the column.

Reply:
Hmm, let's discuss it tomorrow. I'd like to understand what does happen for unmapped values and see if we want to loosen the restriction or have a placeholder mapping with no configuration which feels a bit weird.

Reply:
To capture the decision from our discussion:
- I will ensure that an empty string, null, or undefined value does not use any of the default placeholder strings.
- The existing behavior of showing nothing for unmapped values will be kept.